tests/test_core.py

```diff
@@ -447,6 +447,11 @@ def get_geometry(partition):
     assert_geoseries_equal(result, expected)
 
 
+def test_copy_none_spatial_partitions(geoseries_points):
+    ddf = dask_geopandas.from_geopandas(geoseries_points, npartitions=2)
+    ddf_copy = ddf.copy()
+
```

Comment:
```suggestion
    assert ddf.spatial_partitions is None
    ddf_copy = ddf.copy()
    assert ddf_copy.spatial_partitions is None

```

(not strictly needed, but making it a bit more explicit)

Reply:
yeah that is fair, I've gone with explicitly setting it to None, just in case something happens in between which makes it not None at some point.